nat/interface.go

```diff
@@ -20,13 +20,77 @@ package nat
 // NATService describes fake nat service for darwin
 type NATService interface {
 	Enable() error
-	Add(rule RuleForwarding) error
-	Del(rule RuleForwarding) error
+	Add(rule Rule) error
+	Del(rule Rule) error // TODO remove: replaced by Clear
+	Clear() error
 	Disable() error
 }
 
-// RuleForwarding describes fake nat rule
-type RuleForwarding struct {
-	SourceSubnet string
-	TargetIP     string
+// RuleType specifies what happens to the packets matching the rule
+type RuleType string
+
+const (
+	// Forward matching packets will be accepted and forwarded
+	Forward = RuleType("forward")
```

Comment:
Add RuleType  prefix for each value otherwise nat package will have public nat.Forward, nat.Block which sound like structs or functions.  

```go
RuleTypeForward = RuleType("forward")
RuleTypeBlock = RuleType("block")
```